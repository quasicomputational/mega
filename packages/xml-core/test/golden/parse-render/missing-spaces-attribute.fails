<foo bar="spam"baz="eggs" />
