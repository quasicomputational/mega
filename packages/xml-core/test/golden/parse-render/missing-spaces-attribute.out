Error: Bad syntax: Expected a space before an attribute definition at (1, 16).
